design/ceph/object/store.md

```diff
@@ -119,3 +119,31 @@ Specifying this section also ensures that the pool section in the ceph-object-zo
   zone:
     name: "name"
 ```
+
+### LDAP
+
+The Ceph Object Gateway supports integrating with LDAP for authenticating and creating users, please refer [here](https://docs.ceph.com/docs/master/radosgw/ldap-auth/). This means that the `ceph object store user` is also required to be part of groups in the LDAP server otherwise, authentication will fail. The following settings need to be configured in the RGW server:
```

Comment:
We probably should mention that CA for the x509 cert on ldaps is validated (AFAIK, there isn't a mechanism to disable validation) and a CA bundle will most likely need to be passed in to bind to ldaps.  E.g.

```yaml
kind: CephObjectStore
spec:
  gateway:
    caBundleRef: ldaps-cabundle
```